### ISSUE TITLE
Cant switch to valid key

### ISSUE THREAD

**Issue by coogor** [open-keychain/open-keychain#2474]
I have a key with an expired and an active key - see screenshots attached.
Its not possible to select the valid key for signing/encrypting a mail. OpenKeyChain insists on the expired one.
Sounds more like a bug than a feature to me

## Your Environment

* Android Version: 7.1.1
* Device Model: Galaxy Tab A SM-T555
* OpenKeychain Version: 5.3
* From Google Play or F-Droid?: google

![Screenshot_20190717-075811 jpgokc1](https://user-images.githubusercontent.com/5717703/61354216-fa8fa700-a871-11e9-89c7-11bfd7f3cbeb.jpg)
![Screenshot_20190717-075858](https://user-images.githubusercontent.com/5717703/61354223-fcf20100-a871-11e9-92d4-bc3c7bff199e.jpg)

**Comment by coogor** [comment]
You mean the key with which I signed this expired key? Possible, yes. 
Bit I dont find any way to sign the new key of Mathias (with my new key) - Ideas?

**Comment by dennis-mb** [comment]
The self-signature of the shown key has expired. So it is treated as expired as a whole, even if the expiration date of the encryption subkey is still valid. You won't be able to use that key for message signing and/or encryption.

**Comment by coogor** [comment]
Yes, importing the key again was my idea as well, but the result remains the same - although the key was 'successfully imported'. Note that this key works fine on KMail/GnuPG and a Galaxy S7 Edge using openkeychain as well. Just on the tablet it does not work.
Seems that deleting the key and reimporting it seems to be the only solution?
